private-templates-service/client/src/components/AdaptiveCardPanel/AdaptiveCardPanel.tsx

```diff
@@ -20,7 +20,7 @@ class AdaptiveCardPanel extends React.Component<Props> {
     return (
```

Comment:
Since https://github.com/microsoft/adaptivecards-templates/pull/99 was merged into dev, these changes got added to this PR. I checked our dev website and they work. I believe it's fine if these changes are merged into our stage branch as well.